app/translations/en.json

```
@@ -433,7 +433,7 @@
                     "header": {
                         "noDeployment": "Execute workflow {{workflowName}}",
                         "singleDeployemnt": "Execute workflow {{workflowName}} on {{deploymentId}}",
-                        "singleNamedDeployemnt": "Execute workflow {{workflowName}} on {{deploymentName}} ({{deploymentId}})",
+                        "singleNamedDeployemnt": "Execute workflow {{workflowName}} on {{ name }}",
```

Minor:

To me, `name` sounds a bit ambiguous and I would change it to

```suggestion
                        "singleNamedDeployemnt": "Execute workflow {{workflowName}} on {{ deploymentName }}",
```

and apply the same change across all the files in `en.json` :slightly_smiling_face: